eZ/Publish/Core/Repository/Helper/DomainMapper.php

```diff
@@ -85,12 +90,14 @@ public function __construct(
         ContentHandler $contentHandler,
         LocationHandler $locationHandler,
         TypeHandler $contentTypeHandler,
+        ContentTypeDomainMapper $contentTypeDomainMapper,
```

Comment:
Nitpick: missing PhpDoc param. While at it, could you fix FQCN for FieldTypeRegistry (and add `$fieldtypeRegistry` argument name)? :)

Reply:
fixed in fa9ee61